core/components/atoms/dropdown/__stories__/Options.tsx

```diff
@@ -224,3 +224,16 @@ export const fetchOptions = (searchTerm: string, options = dropdownOptions) => {
     }, 1000);
   });
 };
+
+export const fetchOptionsEmpty = (searchTerm: string, options = []) => {
+  const searchedOptions = searchTerm ? getSearchedOptions(options, searchTerm) : options;
+  return new Promise<fetchOptionSchema>((resolve) => {
+    window.setTimeout(() => {
+      resolve({
+        searchTerm,
+        options: searchedOptions,
```

Comment:
Why do we need to call `getSearchedOptions` function when we can directly pass empty array to `options: []`